lib/filter-headers.js

```diff
@@ -2,15 +2,17 @@
 const ACCEPT_HEADERS = ['accept-language', 'referer', 'user-agent'];
 /**
  * Filter the request headers that are passed to fragment request.
+ * @callback filterHeaders
  *
  * @param {Object} attributes - Attributes object of the fragment node
  * @param {string} attributes.public - Denotes the public fragemnt. Headers are not forward in this case
- * @param {Object} headers - Request header object
+ * @param {Object} request - HTTP Request object
+ * @param {Object} request.headers - request header object
  * @returns {Object} New filtered header object
  */
-module.exports = ({ public: publicFragment }, headers) => 
-    publicFragment 
-        ? {} 
+module.exports = ({ public: publicFragment }, { headers = {} }) =>
```

Comment:
Considering this is an example function, is it worth expanding it a bit to make it really explicit what's happening with the request object being passed in? The style is extremely terse!